tests/precompile/solidity/suites.go

```diff
@@ -12,59 +12,94 @@ import (
 
-	ginkgo.It("tx allow list", ginkgo.Label("Precompile"), ginkgo.Label("TxAllowList"), func() {
-		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-		defer cancel()
+			subnetsSuite.RunHardhatTests(ctx, "tx_allow_list")
+		})
 
-		utils.RunDefaultHardhatTests(ctx, utils.BlockchainIDs["tx_allow_list"], "tx_allow_list")
-	})
+		ginkgo.It("contract deployer allow list", ginkgo.Label("Precompile"), ginkgo.Label("ContractDeployerAllowList"), func() {
+			ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+			defer cancel()
 
-	ginkgo.It("contract deployer allow list", ginkgo.Label("Precompile"), ginkgo.Label("ContractDeployerAllowList"), func() {
-		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-		defer cancel()
+			subnetsSuite.RunHardhatTests(ctx, "contract_deployer_allow_list")
+		})
 
-		utils.RunDefaultHardhatTests(ctx, utils.BlockchainIDs["contract_deployer_allow_list"], "contract_deployer_allow_list")
-	})
+		ginkgo.It("fee manager", ginkgo.Label("Precompile"), ginkgo.Label("FeeManager"), func() {
+			ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+			defer cancel()
 
-	ginkgo.It("fee manager", ginkgo.Label("Precompile"), ginkgo.Label("FeeManager"), func() {
-		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-		defer cancel()
+			subnetsSuite.RunHardhatTests(ctx, "fee_manager")
+		})
 
-		utils.RunDefaultHardhatTests(ctx, utils.BlockchainIDs["fee_manager"], "fee_manager")
-	})
+		ginkgo.It("reward manager", ginkgo.Label("Precompile"), ginkgo.Label("RewardManager"), func() {
+			ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+			defer cancel()
 
-	ginkgo.It("reward manager", ginkgo.Label("Precompile"), ginkgo.Label("RewardManager"), func() {
-		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-		defer cancel()
+			subnetsSuite.RunHardhatTests(ctx, "reward_manager")
+		})
 
-		utils.RunDefaultHardhatTests(ctx, utils.BlockchainIDs["reward_manager"], "reward_manager")
+		// ADD YOUR PRECOMPILE HERE
+		/*
+			ginkgo.It("your precompile", ginkgo.Label("Precompile"), ginkgo.Label("YourPrecompile"), func() {
+				ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+				defer cancel()
+
+				// Specify the name shared by the genesis file in ./tests/precompile/genesis/{your_precompile}.json
+				// and the test file in ./contracts/tests/{your_precompile}.ts
+				// If you want to use a different test command and genesis path than the defaults, you can
+				// use the utils.RunTestCMD. See utils.RunDefaultHardhatTests for an example.
+				subnetsSuite.RunHardhatTests(ctx, "your_precompile")
+			})
+		*/
 	})
-
+}
+
+// Registers the Synchronized Precompile Tests
+// Before running the tests, this function creates the Avalanche locan network without explicitly creating any subnets,
+// and then runs the hardhat tests for each one synchronously.
+/*
+func RegisterSyncTests() {
```

Comment:
Should we remove this block?

Reply:
My intention was to provide a way to show how synced tests can be written. But I guess we don't need that.